app/views/main_layout.scala.html

```diff
@@ -72,7 +72,9 @@
     <!--[if lte IE 8]><link href=' @controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
     <!--[if gt IE 8]><!--> <link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="all" rel="stylesheet" type="text/css" />
         <!--<![endif]-->
+    <script src="https://cdn.optimizely.com/js/21016452515.js"></script>
```

Comment:
this line should now be deleted as the script is now included conditionally below